DemoSwiftApp/AppDelegate.swift

```diff
@@ -82,6 +70,10 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
                 print("Optimizely SDK initiliazation failed: \(error)")
             case .success:
                 print("Optimizely SDK initialized successfully!")
+            #if !os(iOS)
```

Comment:
what's the need of this check.

Reply:
Other platforms (tvOS and watchOS) give warning to ask to add default. To clean up warning.